bucket/kawaii-player.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "5.0.0-1",
+    "description": "Audio/Video manager and mutlimedia player (based on mpv and mplayer) with PC-To-PC casting feature, along with functionalities of portable media server and torrent streaming server.",
+    "homepage": "https://github.com/kanishka-linux/kawaii-player",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kanishka-linux/kawaii-player/releases/download/v5.0.0-1/kawaii-player-5.0.0-1.windows10.x86_64-bin.7z",
+            "hash": "7249072cdd4914a0da276fe1c2990579df486555dce5dcc6cd6f889c22e90864",
+            "extract_dir": "kawaii-player-5.0.0-1.windows10.x86_64-bin"
+        }
+    },
+    "bin": "kawaii_player.exe",
```

Comment:
Does the app support command-line **arguments**. We only add `bin` for GUI apps if they support command-line args.

Reply:
There's not any arguements but we can launch it using terminal

Reply:
> There's not any arguements but we can launch it using terminal

That doesn't match the criteria of using `bin`. Please remove `bin`.